src/js/common/containers/MHVApp.jsx

```diff
@@ -91,33 +93,28 @@ export class MHVApp extends React.Component {
       return <AcceptTermsPrompt terms={terms} cancelPath="/health-care/" onAccept={this.props.acceptTerms}/>;
     }
 
-    if (!this.isAccessible()) {
+    if (!this.hasService()) {
       return mhvAccessError;
     }
 
-    const enabled = this.props.isDataAvailable === true || typeof this.props.isDataAvailable === 'undefined';
-
-    const view = React.Children.map(this.props.children,
-      (child) => {
-        let props = null;
-        if (typeof child.type === 'function') {
-          props = { isDataAvailable: enabled };
-        }
-        return React.cloneElement(child, props);
-      }
-    );
```

Comment:
We've determined previously that this `isDataAvailable` prop isn't being propagated properly due to the `DowntimeNotification` component not propagating it one level above. It doesn't matter anyway because this was related to doing the check for the services within `RequiredLoginView`, but we are checking that and handling it in `MHVApp` for these apps.